test/fastifyAwilixPlugin.dispose.spec.js

```diff
@@ -39,11 +39,11 @@ describe('fastifyAwilixPlugin', () => {
   })
 
   variations.forEach((variation) => {
-    describe(variation.name, () => {
+    describe(`${variation.name}`, () => {
```

Comment:
`variation.name` is `string`, wrapping do not make any difference.

```suggestion
    describe(variation.name, () => {
```

Reply:
Cheers @climba03003, have remove that commit